docs/get-started/index.md

```diff
@@ -250,123 +242,4 @@ were previously applied from that kustomization.
 If you alter the webapp deployment using `kubectl edit`, the changes will be reverted to match
 the state described in git. When dealing with an incident, you can pause the reconciliation of a
 kustomization with `flux suspend kustomization <name>`. Once the debugging session
-is over, you can re-enable the reconciliation with `flux resume kustomization <name>`.
-
-## Production bootstrap 
```

Comment:
Per our conversation, I would suggest renaming this section Multiple Clusters in Production and keeping it. Then, later, we can do further improvements for this section.

Reply:
Cross-linking https://github.com/fluxcd/flux2/issues/611